src/ResourceManagement/BatchAI/ContainerImageSettingsImpl.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.BatchAI.Fluent.Models;
+
+namespace Microsoft.Azure.Management.BatchAI.Fluent
+{
+    using Microsoft.Azure.Management.BatchAI.Fluent.Models;
+    using Microsoft.Azure.Management.BatchAI.Fluent.BatchAIJob.Definition;
+    using Microsoft.Azure.Management.BatchAI.Fluent.ContainerImageSettings.Definition;
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core.ChildResource.Definition;
+
+///GENTHASH:Y29tLm1pY3Jvc29mdC5henVyZS5tYW5hZ2VtZW50LmJhdGNoYWkuaW1wbGVtZW50YXRpb24uQ29udGFpbmVySW1hZ2VTZXR0aW5nc0ltcGw=
+    internal partial class ContainerImageSettingsImpl  :
+        IndexableWrapper<Microsoft.Azure.Management.BatchAI.Fluent.Models.ImageSourceRegistry>,
+        IDefinition<BatchAIJob.Definition.IWithCreate>
+    {
+        private BatchAIJobImpl parent;
+
+        ///GENMHASH:E69EAFBE5FEE03FDD5408C54C9FE8D17:CA53B7BE62B1D1854755EDD010D28892
+        protected  ContainerImageSettingsImpl(ImageSourceRegistry inner, BatchAIJobImpl parent) : base(inner)
+        {
+            this.parent = parent;
+        }
+
+        ///GENMHASH:077EB7776EFFBFAA141C1696E75EF7B3:7EF63673EC66A608A59744FDA0DA741A
+        public IWithCreate Attach()
```

Comment:
Please revise entire file. It does not seems to be implemented properly.

Reply:
Fixed.